### DISCUSSION TITLE
Variation by weight - Link Weight to sku/variant

### DISCUSSION THREAD

**Discussion by kiernangs** [etsy/open-api#1078] [Ideas]
Currently etsy only has a single weight per listing.  
------------
The etsy listings allows product variations to vary in price, including setting the price variable to be different for every single sku  (2 variations, everything varies by price).

Allowing weight to also be assigned as a variable in this same way as price would fix a LOT of issues with calculated shipping, especially for international orders.

-------------
In any given listing, products are often offered in Small, Medium, Large, etc.  These sizes don't make sense to each have their own listing, but clearly each have their own weight.

**Comment by kiernangs** [comment]
I believe this could be similar to how sku_on_property  is described here: https://developers.etsy.com/documentation/tutorials/listings/#updating-inventory

But weight_on_property

**Comment by kiernangs** [comment]
I wonder if sometime in the next decade this will be considered by etsy.

So far, I see sellers requesting this feature as far back as 11 years ago on the forums.  I know I have asked etsy a few times over the last 10 years.

Take bets on etsy caring at all?  ...depressing

**Reply by Gareth064** [reply]
Don't hold your breath on this one bud. This sounds like a change to the core application. The Etsy developers who sometimes look in here (which is rare nowadays) are from the API team only and have nothing to do with the core product.

I have no idea how you would raise these types of requests with Etsy. Maybe emailing developer@etsy.com will get the request to someone.

**Reply by kiernangs** [reply]
Yeah... "taking bets on etsy caring" is  in the intial post for that reason.  They don't seem to understand how big an issue it is somehow.

**Comment by kiernangs** [comment]
Anyone else have tips on getting etsy developers to care about sellers at all?

**Comment by kiernangs** [comment]
@etsyachristensen   Its been a decade or more since this was first requested, and it shows up as  a near constant issue on the etsy forums. 
Any chance it ever gets resolved?

**Reply by turboturtleD** [reply]
He is no longer working for Etsy.

**Reply by kiernangs** [reply]
That's typical. One of the few people from Etsy to actually engage anyone here....and they leave. 

Yeah that's Etsy
